src/main/lib/bundle/bundlers/bundler.ts

```diff
@@ -11,6 +11,11 @@ const readFileAsync = util.promisify(readFile);
 const writeFileAsync = util.promisify(writeFile);
 const unlinkAsync = util.promisify(unlink);
 
+/**
+ * A
```

Comment:
What is that comment?

Reply:
It's an unfinished comment, that's what it is... The embarassment.